packages/block-library/src/search/edit.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+
+export default function SearchEdit( { className, attributes, setAttributes } ) {
+	const { label, placeholder, buttonText } = attributes;
+
+	return (
+		<div className={ className }>
+			<RichText
+				wrapperClassName="wp-block-search__label"
+				value={ label }
+				placeholder={ __( 'Add label' ) }
+				keepPlaceholderOnFocus
+				formattingControls={ [] }
+				onChange={ ( html ) => setAttributes( { label: html } ) }
+			/>
+			<input
```

Comment:
Should this also be a RichText? I suppose it's ok as an input.

Either way, I think it should have an `aria-label`, as placeholders aren't considered a replacement for labels.

Reply:
Line breaks and HTML aren't supported in the `placeholder` attribute so I went with `<input>` here instead of `PlainText` or `RichText`.

Reply:
Added an `aria-label` in 1eae8ea6c! 👌